cunumeric/runtime.py

```diff
@@ -96,6 +97,24 @@ def _register_dtypes(self):
         for numpy_type, core_type in _supported_dtypes.items():
             type_system.make_alias(np.dtype(numpy_type), core_type)
 
+        for n in range(1, LEGATE_MAX_DIM + 1):
+            self._register_point_type(n)
+
+    def _register_point_type(self, n):
+        type_system = self.legate_context.type_system
+        point_type = "" + str(n)
```

Comment:
Give this a better identifier, e.g. "Point" + str(n), not just a number.

Reply:
Yes, I should have done that!